### PR TITLE
Add `kd machine cp` command

### PR DESCRIPTION
**Pull request by ppknap**
This PR adds `machine cp` subcommand to the new architecture.

Fixes: #10681
Implements: #10615, #8458

## Motivation and Context
Lack of `machine cp` command in kd

## How Has This Been Tested?
Unit tests, Manually

## Screenshots (if appropriate):
https://asciinema.org/a/2nkxu1n19kgr4n08mcfc8x9mk

## Types of changes
- [x] New feature (non-breaking change which adds functionality)

:warning: This PR looks big because of renamed `fs.go -> fsold.go` and `fs_test.go -> fsold_test.go` files. They didn't change, git detected them as rename(they have history) however github doesn't show them as that.

### CODE REVIEW

File: go/src/koding/klient/fs/fs.go [hunk 1]
```diff
@@ -1,418 +1,51 @@
-// Package fs provides file system handleFuncs that can be used with kite
```
Comment (ppknap) [inline go/src/koding/klient/fs/fs.go]: This file contains new code. Please review me.

File: go/src/koding/klient/fs/fs_test.go [hunk 2]
```diff
@@ -1,1079 +1,101 @@
-package fs
```
Comment (ppknap) [inline go/src/koding/klient/fs/fs_test.go]: This file contains new code.

File: go/src/koding/klient/fs/fsold.go [hunk 3]
```diff
@@ -0,0 +1,418 @@
+// Package fs provides file system handleFuncs that can be used with kite
```
Comment (ppknap) [inline go/src/koding/klient/fs/fsold.go]: This file is a rename - please skip it.

File: go/src/koding/klient/fs/fsold_test.go [hunk 4]
```diff
@@ -0,0 +1,1079 @@
+package fs
```
Comment (ppknap) [inline go/src/koding/klient/fs/fsold_test.go]: This file is a rename - please skip it.

File: go/src/koding/kites/kloud/klient/klient.go [hunk 5]
```diff
@@ -368,6 +369,25 @@ func (k *Klient) CurrentUser() (string, error) {
...
+
+	raw, err := k.Client.TellWithTimeout("fs.abs", k.timeout(), req)
+	if err != nil {
+		return "", false, false, err
+	}
+
+	var res fs.AbsResponse
+	if err := raw.Unmarshal(&res); err != nil {
+		return "", false, false, err
+	}
```
Comment (rjeczalik) [inline go/src/koding/kites/kloud/klient/klient.go]: You could use call method instead, reduces boilerplate.

**Review by rjeczalik — APPROVED** [review]

**Comment by rjeczalik** [comment]
Is it still wip?

**Comment by ppknap** [comment]
@rjeczalik yes I'm adding tests and want do improve progress status.

File: go/src/koding/klient/app/klient.go [hunk 6]
```diff
@@ -777,7 +777,7 @@ func (k *Klient) Run() {
 
 	// TODO(rjeczalik): Enable after TMS-848.
 	if k.autoupdateEnabled() {
-		go k.updater.Run()
+		//go k.updater.Run()
```
Comment (ppknap) [inline go/src/koding/klient/app/klient.go]: @rjeczalik this is wip I made this change temporary in order to use it on my remote machine for testing. Will remove this after all the code is done.

Comment (rjeczalik) [inline go/src/koding/klient/app/klient.go]: My bad, sorry.

File: go/src/koding/klient/fs/kite.go [hunk 7]
```diff
@@ -0,0 +1,30 @@
+package fs
+
+import (
+	"github.com/koding/kite"
+)
+
+// KiteHandlerAbs creates a kite handler function that, when called, invokes
+// default file system Abs method.
+func KiteHandlerAbs() kite.HandlerFunc {
+	return func(r *kite.Request) (interface{}, error) {
```
Comment (rjeczalik) [inline go/src/koding/klient/fs/kite.go]: Why you need a closure here? What about just:

```go
type KiteHandlerAbs(r *kite.Request) (interface{}, error) { ...
```
```

**Review by rjeczalik — APPROVED** [review]

**Review by cihangir — APPROVED** [review]

**Pull request merged**